### ISSUE TITLE
Pitest support issue

### ISSUE THREAD

**Issue by alvenchen999** [pitest/pitest-junit5-plugin#79]
Does pitest support bazel build?
Does pitest commandline tool support junit5?

I go through the org page that it looks like junit5 only supported by the maven plugin? pls help to correct me, thx

**Comment by szpak** [comment]
> I go through the org page that it looks like junit5 only supported by the maven plugin? pls help to correct me, thx

No, being on the classpath the plugin should be automatically detected. If in trouble, you can take a look at the Gradle plugin which uses the "command line interface", but AFAIR there was no extra magic to make it work.
